gobblin-metastore/build.gradle

```diff
@@ -77,6 +77,9 @@ artifacts {
 
 test {
     workingDir rootProject.rootDir
+    if (project.hasProperty('usePreinstalledMysql') && Boolean.parseBoolean(project.usePreinstalledMysql)) {
+      jvmArgs '-Dgobblin.metastore.testing.embeddedMysqlEnabled=false'
```

Comment:
This is only for Travis. The container sets up a mysql instance with the passwordless root user.